zh-CN/mapping_dev.zh_CN

```diff
@@ -386,21 +386,21 @@ dlgSgBranchAddCheckoutOverwriteExisting.fur=单击 “取消” 以选择其他
 dlgSgBranchAddCheckoutOverwriteExisting.hdl%1=分支 “$1” 已经存在。你想覆盖它吗？
 dlgSgBranchAddCheckoutOverwriteExisting.tle=创建分支
 dlgSgBranchDeleteLocalConfirmMultiple.btn"Delete"=删除
-dlgSgBranchDeleteLocalConfirmMultiple.chk"Delete from remote repository"=从远程仓库中删除
+dlgSgBranchDeleteLocalConfirmMultiple.chk"Delete from remote repository"=从 remote 仓库中删除
```

Comment:
emmm, I think the "remote branch" should not be changed. 

In the git workflow configuration, 
"Master Branch", etc., these are configuration prompts,
so I think it can be retained to avoid people not knowing after the translation, 
because like "主" "开发" "支持" "功能"  There is no original English conspicuous.
but there is nothing special about "remote", so it should be no problem to translate into Chinese.

It just [these lines](https://github.com/syntevo/smartgit-translations/pull/12#discussion_r303193555) need to be keeped.

After fixed this. this pull request could be merged.

Reply:
Cheers,  I've fixed it.